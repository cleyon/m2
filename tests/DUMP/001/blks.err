Block # 7, Type=AGG: (blk_dump_blktab)
  count   : 0

Block # 4, Type=AGG: (blk_dump_blktab)
  count   : 0

Block # 8, Type=AGG: (blk_dump_blktab)
  count   : 0

Block # 5, Type=AGG: (blk_dump_blktab)
  count   : 0

Block # 6, Type=AGG: (blk_dump_blktab)
  count   : 0

Block # 9, Type=AGG: (blk_dump_blktab)
  count   : 0

Block # 11, Type=FILE: (blk_dump_blktab)
  filename: ./.m2rc
  open    : False
  atmode  : ProcessAt
  dstblk  : 0
Block # 10, Type=FILE: (blk_dump_blktab)
  filename: /Users/cleyon/.m2rc
  open    : False
  atmode  : ProcessAt
  dstblk  : 0
Block # 12, Type=FILE: (blk_dump_blktab)
  filename: blks.m2
  open    : True
  atmode  : ProcessAt
  dstblk  : 0
Block # 1, Type=AGG: (blk_dump_blktab)
  count   : 0

Block # 2, Type=AGG: (blk_dump_blktab)
  count   : 0

Block # 3, Type=AGG: (blk_dump_blktab)
  count   : 0

