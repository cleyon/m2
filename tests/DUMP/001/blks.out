BROKEN: (blk_dump_blktab)
